src/main/java/org/jabref/logic/formatter/bibtexfields/ReplaceTabsBySpaceFormater.java

```diff
@@ -9,13 +9,13 @@
 /**
  * Replaces any tab with a space
  */
-public class RemoveTabsFormatter extends Formatter {
+public class ReplaceTabsBySpaceFormater extends Formatter {
 
     private static final Pattern TAB = Pattern.compile("\t+");
 
     @Override
     public String getName() {
-        return Localization.lang("Remove tabs");
+        return Localization.lang("Replace tabs with space");
```

Comment:
Could we also use `by` instead of `with`? 😇  (consistency with class name)

Reply:
Yep - will change now!